### PR TITLE
Writing tests

### PR DESCRIPTION
**Pull request by BrandyMcKinney**
- Tests were written for every function in the PersonalAssistant class. These test include markers for components and smoke tests. 
- Create `pytest.ini` file to register markers 
- Edited README to include running pytest with various flags
- Wrote and refactored code to pass all tests

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[pytest]
+markers =
+  tasks_component: marks test as associated with tasks (deselect with '-m "not tasks"').
```
Comment (duepayer) [inline pytest.ini:3]: nit: I'd have named these simply tasks, birthdays and contacts but nothing wrong with being extra explicit with your `*_component` naming convention

File: test_personal_assistant.py [hunk 2]
```diff
@@ -45,92 +46,110 @@ def test_add_task_invalid_type(assistant):
           \nEXCTYPE: {excinfo.type} \
           \nEXCTRACEBACK: {excinfo.traceback}\n")
 
-# Test remove_task
-
 
+# Test remove_task
+@pytest.mark.tasks_component
```
Comment (duepayer) [inline test_personal_assistant.py:51]: I'd add a smoke marker here because all basic CRUD functionality should be covered.

**Review by duepayer — APPROVED** [review]
Pulled down files and ran locally and everything looked good. Left a few comments about nice to have but not necessary changes so this is good to go! Great job!

File: test_personal_assistant.py (hunk 2 above)
Comment (BrandyMcKinney) [reply test_personal_assistant.py:51]: Ok!I added smoke markers to the all of the delete methods.

**Pull request merged**